txscript/scriptnum.go

```diff
@@ -197,13 +197,6 @@ func (n ScriptNum) Int32() int32 {
 // bytes and therefore will pass that value to this function resulting in an
 // allowed range of [-2^31 + 1, 2^31 - 1].
 //
-// The requireMinimal flag causes an error to be returned if additional checks
```

Comment:
Well, I already approved it, but taking a second look, I wonder if it should mostly be left, just updated to something like: 

```Go
// An error will be returned if the encoding is not represented with the
// smallest possible number of bytes or is the negative 0 encoding, [0x80].  For
// example, consider the number 127.  It could be encoded as [0x7f], [0x7f
// 0x00], [0x7f 0x00 0x00 ...], etc.  All forms except [0x7f] will return an
// error.
```

Reply:
Yes, still relevant.  I'll make that change

Reply:
Might also want to fix the formatting on the warning text as well.  Blank line before and there is an extra space after the colon.

e.g.

```Go
//
// WARNING: Great care should be taken if passing a value larger than
// defaultScriptNumLen, which could lead to addition and multiplication
// overflows.
```

Reply:
Also fixed the out-of-date reference to `ErrNumberTooBig` that was renamed to `ErrNumOutOfRange` in d8306ee60205e682885234b4e360b1e59014765a

Reply:
Aye, I saw that.  Nice catch!